packages/cli/src/commands/build.ts

```diff
@@ -434,7 +457,7 @@ async function doBuild(
   // Execute Builders for detected entrypoints
   // TODO: parallelize builds (except for frontend)
   const sortedBuilders = sortBuilders(builds);
-  const buildResults: Map<Builder, BuildResult> = new Map();
+  const buildResults: Map<Builder, BuildResult | BuildOutputConfig> = new Map();
```

Comment:
Because we add a `BuildOutputConfig` object to the `buildResults` map later in this code, we have to expand the allowed types here.

This had a couple of knock-on effects on related functions, but the type overlaps work out just fine.